_pages/hiring-and-jobs/hiring.md

```diff
@@ -8,108 +8,47 @@ questions:
-  - If yes, what will the release date be (note that the effective date must fall on a [payroll period](https://www.gsa.gov/buying-selling/purchasing-programs/shared-services/payroll-shared-services/payroll-calendars))
-  - Be aware that the candidate may be on a project and have to finish out/reassign their work so may not be able to start on your team as soon as you’d like. It is up to their current supervisor when they are willing to let them move to your team.
-- Before moving forward with a selection, Talent recommends conducting internal reference checks. Feel free to refer to [this template](https://docs.google.com/document/d/1QL6711NSv3nsIlN_3Rc7DFk8Apubyy8ugQHAAEWRWgc/edit) for guidance.
-
-#### Job prep
-
-Use the following templates to prepare the posting (make sure to make a copy of each template):
-
-- [TTSJobs Announcement Template](https://docs.google.com/document/d/1YIliZcF8dhqs4GzBAKYj5niqdgcN4tggTTDl3zeIOO8/edit?usp=sharing)
-- [TTSJobs Direct Application Form Template](https://docs.google.com/forms/d/1kpsTvpfCBox7ksQtX9VjQQdjkflPGCdKIZK-a9nUDPU/edit)
-- [TTSJobs Application & Nomination Form Template](https://docs.google.com/forms/d/1C8OVwj-yvuHicDwPbtFTLxPLn8zZA6WK6HC-0uIzHwk/edit)
-- [Template - TTS Interview Guide](https://docs.google.com/document/d/1GCVgMUBS-Hi7STLdWO6Ci47LXbYCz85U2kpVA71AZLc/edit#heading=h.6jehpcfzn58u)
-  - An interview guide for the role being advertised may alredy exist if the role was previously posted externally. Reach out to jointts@gsa.gov and they can check for you.
-
-Once the TTSJobs Announcement is ready, Talent will:
-
-- Review the job announcement and interview guide: email [jointts@gsa.gov](mailto:jointts@gsa.gov) and a team member will review
-- Add it to [#tts-jobs](https://gsa-tts.slack.com/messages/tts-jobs/) on Slack informing everyone of the new opportunity
-- Post it to [TTSJobs Listings]({{site.baseurl}}/ttsjobs/#announcements)
-
-#### Qualification
+- [Fair & Equitable Hiring Practices]({{site.baseurl}}/fair-and-equitable-hiring-practices)
 
-In addition to the information within the templates, the hiring manager will need to manage the following:
+## Promotions
 
-- Developing the interview process
-- Scheduling interviews
-- Making a selection
-- filling out the [candidate pool template](https://docs.google.com/spreadsheets/d/1EN3iLUmmDQ4iX5k-AsDsUPUd_igrEy3BEtlIs5KM59w/edit?usp=sharing)
-- Letting the Talent Team know who was selected for the position
+“Promotions” in the traditional, private sector sense do not exist within the federal government. The [Promotions Handbook page]({{site.baseurl}}/promotions) explains Merit Promotions for both Competitive Service (Career) and Excepted Service positions.
 
-The hiring team is responsible for confirming that applicants meet the qualifications outlined in the posting (i.e. GS level, required experience). For HR to move forward with a selection, the applicant must meet the following qualifications:
+## TTS Hiring Process
```

Comment:
I may be getting lost (this is a very large PR) but wasn't this already covered? Or does it just seem repetitive because there's a lot of overlap in the processes? I'm wondering if we have an opportunity to tighten this up some (maybe in a follow-up PR)

Reply:
This page is the overview for where all other information lives, so some small overlap here.